src/styles/blocks/contacts.scss

```diff
@@ -0,0 +1,33 @@
+.contacts {
+  &__info {
+    margin: 0 0 24px;
+  }
+
+  &__info-tittle {
+    margin: 0 0 8px;
+    font-size: 14px;
+    line-height: 20px;
+  }
+
+  &__info-link {
+    font-weight: 500;
+    text-decoration: none;
+    color: inherit;
+  }
```

Comment:
Add hovers
#4 to the checklist:
Change text color on hover for phone, email, and address.

Reply:
Open the address in the new tab
#9 Location-related addresses/links should open google maps in a new tab target="_blank"